tests/inventory.yml

```diff
@@ -6,3 +6,30 @@ vagrant:
       ansible_python_interpreter: /usr/bin/python3
       ansible_ssh_private_key_file: ~/.ssh/ota-vagrant
       ota_engine_config_path: ./engine_config.json
+      ota_engine_ota_bot_private_key: !vault |
```

Comment:
Do we really want to use the proper private key in tests? As I understand it, this prevents contributors from running their own tests, and it risks accidental publishing when running tests.

Reply:
Added a note in the Test section in the README